modules/ROOT/pages/kubernetes-hardening-guide.adoc

```diff
@@ -0,0 +1,61 @@
+= Kubernetes Hardening Guide
+
+The following guide describes the security challenges associated with setting up and securing a
+Kubernetes cluster based on the guidelines provided by the NSA (National Security Agency). For complete guidance, see the https://media.defense.gov/2022/Aug/29/2003066362/-1/-1/0/CTR_KUBERNETES_HARDENING_GUIDANCE_1.2_20220829.PDF[Kubernetes Hardening Guide^].
+
+== Common Sources of Compromise in K8s
+
+Review the following common sources of compromise:
+
+* Supply chain risks:
+** Container build cycle
+** Infrastructure acquisition
+* Malicious threat actors:
+** Exploit vulnerabilities and misconfigurations
+* Insider threats:
+** Administrators, users or cloud providers abusing their privileges
+
+== Mitigations
+
+Review the following mitigations:
+
+* Scan containers and pods for vulnerabilities or misconfigurations.
+* Run containers and pods with the least privileges possible.
+* Use network separation to control the amount of damage a compromise can cause.
+* Use firewalls to limit unneeded network connectivity and use encryption to protect confidentiality.
+* Use strong authentication and authorization to limit user and administrator access as well as to limit the attack surface.
+* Capture and monitor audit logs so that administrators can be alerted to potential malicious activity.
+* Periodically review all K8s settings and use vulnerability scans to ensure risks are appropriately accounted for and security patches are applied.
+
+== Recommendations and Implementation
```

Comment:
Can we somehow add a paragraph stating that we are following this practices on our deployments as well, is not only recommendations for customers but also what we also follow on our own deployments.